app/models/search_result.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class SearchResult < ApplicationRecord
+  enum status: { processing: 0, completed: 1 }
+end
```

Comment:
Should this be implemented in #19 instead? 🙏

Reply:
@Tuubz 🙆‍♂️